front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -397,6 +420,7 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
               className="btn btn-warning"
               type="button"
               onClick={updateStudy}
+              disabled={!areEndDatesValid}
```

Comment:
Should there be a message for the user in case he doesn't understand why he can't validate the form ?

Reply:
after a discussion with the ux designer, we've made it impossible to select incorrect dates, so there's no need for messages or to disable the button